src/www/widgets/widgets/picture.widget.php

```diff
@@ -76,7 +76,7 @@
       </tr>
       <tr>
         <td>
-          <input id="submita" name="submita" type="submit" class="btn btn-primary formbtn" value="<?= gettext('Upload') ?>" />
+          <input id="submitb" name="submitb" type="submit" class="btn btn-primary formbtn" value="<?= gettext('Upload') ?>" />
```

Comment:
can you please make these a bit more specific to their use case, e.g. "submit_widget"